package.json

```diff
@@ -42,19 +42,19 @@
     "eslint-plugin-node": "^8.0.1",
     "eslint-plugin-prettier": "^3.0.1",
     "eslint-plugin-promise": "^4.0.1",
-    "fastify": "^2.0.0",
-    "hemera-plugin": "^2.0.0",
+    "fastify": "^3.2.0",
+    "hemera-plugin": "^2.0.2",
     "hemera-testsuite": "^4.1.0",
-    "light-my-request": "^3.2.0",
+    "light-my-request": "^4.0.1",
     "np": "^4.0.2",
     "pre-commit": "^1.2.2",
     "prettier": "^1.16.4",
     "tap-spec": "^5.0.0"
```

Comment:
could be removed... but generally I don't mess with given options other than incremental :)

Reply:
Feel free to remove it.

Reply:
removed